Bottom: 41d8934c86c931a8568293264d1b3c6b80101666
Top:    457d34b04f8bf7d8314a3ea0b43ed65c321c8b32
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-19 00:53:23 +0000

find.c: Make some setup / teardown functions not used outside of find.c static.

This makes the following functions private to find.c:

  InitComponentLookup
  FreeComponentLookupMemory
  InitLayoutLookup
  FreeLayoutLookupMemory


---

diff --git a/src/find.c b/src/find.c
index 2d8871f..910e629 100644
--- a/src/find.c
+++ b/src/find.c
@@ -539,7 +539,7 @@ PV_TOUCH_PV (PinType *PV1, PinType *PV2)
 /* ---------------------------------------------------------------------------
  * releases all allocated memory
  */
-void
+static void
 FreeLayoutLookupMemory (void)
 {
   Cardinal i;
@@ -559,7 +559,7 @@ FreeLayoutLookupMemory (void)
   RatList.Data = NULL;
 }
 
-void
+static void
 FreeComponentLookupMemory (void)
 {
   free (PadList[0].Data);
@@ -572,7 +572,7 @@ FreeComponentLookupMemory (void)
  * allocates memory for component related stacks ...
  * initializes index and sorts it by X1 and X2
  */
-void
+static void
 InitComponentLookup (void)
 {
   Cardinal i;
@@ -606,7 +606,7 @@ InitComponentLookup (void)
  * allocates memory for component related stacks ...
  * initializes index and sorts it by X1 and X2
  */
-void
+static void
 InitLayoutLookup (void)
 {
   Cardinal i;
diff --git a/src/find.h b/src/find.h
index de0a8f8..9644727 100644
--- a/src/find.h
+++ b/src/find.h
@@ -56,11 +56,7 @@ bool ResetFoundLinesAndPolygons (bool, int flag);
 bool ResetFoundPinsViasAndPads (bool, int flag);
 bool ResetConnections (bool, int flag);
 void InitConnectionLookup (void);
-void InitComponentLookup (void);
-void InitLayoutLookup (void);
 void FreeConnectionLookupMemory (void);
-void FreeComponentLookupMemory (void);
-void FreeLayoutLookupMemory (void);
 void RatFindHook (int, void *, void *, void *, bool, int flag, bool);
 int DRCAll (void);
